src/main/java/ca/uhn/fhir/jpa/starter/service/HelperService.java

```diff
@@ -1627,6 +1706,168 @@ private <R extends IBaseResource> String updateResource(String keycloakId, IBase
 		return null;
 	}
 
+	private <R extends IBaseResource> String createResource(String keycloakId, IBaseResource resource, Class<R> resourceClass) {
+
+		RealmResource realmResource = FhirClientAuthenticatorService.getKeycloak().realm(appProperties.getKeycloak_Client_Realm());
+		R existingResource = null;
+		GroupResource groupResource = null;
+		GroupRepresentation group = null;
+		try {
+			if (resourceClass.equals(Organization.class)){
+				groupResource = realmResource.groups().group(keycloakId);
+				group = groupResource.toRepresentation();
+				existingResource = FhirClientAuthenticatorService.getFhirClient().read().resource(resourceClass).withId(group.getAttributes().get("organization_id").get(0)).execute();
```

Comment:
`group.getAttributes().get("organization_id").get(0))` assign this to a variable and use the variable in withId function for better clarification.